Makefile

```diff
@@ -142,6 +142,7 @@ examples:
 	@$(call label,Building examples)
 	@echo
 	$(ECHO_V)$(MAKE) -C examples/keyvalue ECHO_V=$(ECHO_V)
+	$(ECHO_V)go test ./examples/simple
```

Comment:
why not follow the same format as keyvalue? shouldn't we run the same commands to test both?